python/cugraph/structure/graph_implementation/npartiteGraph.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .simpleGraph import simpleGraphImpl
+import cudf
+
+
+class npartiteGraphImpl(simpleGraphImpl):
+    def __init__(self, properties):
+        super(npartiteGraphImpl, self).__init__(properties)
+        self.properties.bipartite = properties.bipartite
+
+    # API may change in future
+    def __from_edgelist(
+        self,
+        input_df,
+        source="source",
+        destination="destination",
+        edge_attr=None,
+        renumber=True,
+    ):
+        self._simpleGraphImpl__from_edgelist(
+            input_df,
+            source=source,
+            destination=destination,
+            edge_attr=edge_attr,
+            renumber=renumber,
+        )
+
+    def sets(self):
+        """
+        Returns the bipartite set of nodes. This solely relies on the user's
```

Comment:
That is because we don't want to expose or let user use n-partite Graphs yet. 
The doc mentions only the return as it doesn't have any arguments/parameters